gulpfile.js

```diff
@@ -45,9 +47,27 @@ const g3w         = require('./config');
 let production   = false;
 let outputFolder = g3w.admin_overrides_folder;
 
+// ANSI color codes
+const INFO__ = "\033[0;32m\#\#\# ";
+const __INFO = " \#\#\# \033[0m";
+const H1__ = "\n\n" + INFO__;
+const __H1 = __INFO + "\n";
+
+
 // Retrieve project dependencies ("g3w-client")
 const dependencies = Object.keys(packageJSON.dependencies).filter(dep => dep !== 'vue');
 
+// Built-in client plugins
+const default_plugins = [
+  'editing',
+  'qplotly',
+  'qtimeseries'
+];
```

Comment:
@volterra79 should I also include client `openrouteservice` as built-in plugin?

**Ref:**

- [g3w-admin/openrouteservice](https://github.com/g3w-suite/g3w-admin/blob/5e4e294a91bd24fe4cfa0e834d067c7016c64cf3/g3w-admin/openrouteservice)
- [g3w-client-plugin-openrouteservice](https://github.com/g3w-suite/g3w-client-plugin-openrouteservice)